index.js

```diff
@@ -77,27 +77,34 @@ exports.login = async function (options) {
 
         // attempt to capture accountId automatically via browser callback
         let tempUrl;
-        // See: https://github.com/near/near-shell/issues/358
-        const isMac = process.platform === 'darwin';
+        const isWin = process.platform === 'win32';
 
         // find a callback URL on the local machine
         try {
-            // capture callback isn't working reliably outside of Mac. This is a workaround to not use it
-            if (isMac) {
+            if (!isWin) { // capture callback is currently not working on windows. This is a workaround to not use it
                 tempUrl = await capture.callback(5000);
-                // if we found a suitable URL, attempt to use it
-                newUrl.searchParams.set('success_url', `http://${tempUrl.hostname}:${tempUrl.port}`);
-                await openUrl(newUrl);
-            } else {
-                // redirect automatically, but do not use the browser callback
-                await openUrl(newUrl);
             }
         } catch (error) {
             // console.error("Failed to find suitable port.", error.message)
             // TODO: Is it? Try triggering error
             // silent error is better here
         }
 
+        // if we found a suitable URL, attempt to use it
+        if (tempUrl) {
```

Comment:
we can move all this logic into the above `try catch` under `if (!isWin)` but I'm fine with it

Reply:
@mikedotexe I just reverted previous change in this part of code.